config/default/translations/messages-en.properties

```diff
@@ -498,3 +498,5 @@ report.undo_death_report.undo.undo_information = Undo death report
 
 tour.analytics.overview.description = The Targets tab shows your key performance indicators. For each target, the app displays your progress this month and your goal for this month. Progress numbers will be calculated automatically by the app, and goals will be set by your organization. Your manager and support team will also be able to view your indicators.
 tour.analytics.overview.title = Targets tab
+
+messages.generic.sys.facility_not_found = You have not been registered to use this app
```

Comment:
We should be adding this to the main translations, here: https://github.com/medic/cht-core/blob/master/ddocs/medic/_attachments/translations/messages-en.properties instead of the config translations. 
Also, please add it in the appropriate spot in the file (alphabetically).